packages/xo-web/src/common/xo/index.js

```diff
@@ -848,7 +848,7 @@ export const restartHostAgent = async host => {
         ),
         title: _('restartHostAgent'),
       })
-      return _call('host.restart_agent', { id: resolveId(host), ignoreBackup: true })
```

Comment:
I see more occurences of `ignoreBackup` in the code.

Reply:
I think these occurences are local variable

Reply:
At the time of my comment, only this line has been changed and I do believe the other changes are necessary as well.